gcc/testsuite/gfortran.dg/dec_intrinsic_int_real_promotion-2.f

```diff
@@ -0,0 +1,114 @@
+! { dg-do compile }
+! { dg-options "-fdec" }
+!
+! Test promotion between integers and reals in intrinsic operations.
+! These operations are: mod, modulo, dim, sign, min, max, minloc and
+! maxloc.
+!
+      PROGRAM promotion_int_real
+        REAL l/0.0/
+        LOGICAL a_i
```

Comment:
Hi, sorry for being nitpicking about notation, but in the test which you have use as template for this one https://github.com/CodethinkLabs/gcc/blob/codethink/gcc-7_3_0-release-164-g72e689b-1/gcc/testsuite/gfortran.dg/dec_intrinsic_int_real_promotion.f I have been following a notation which would identify the variable type when used in the intrinsic funtions:
- for INTEGERS I've added "_i"  suffix to the variable name
- for REALS I've added "_r" suffix to the variable name

It would be nice and recommendable if we continue with this notation in dec_intrinsic_int_real_promotion-2.f and dec_intrinsic_int_real_promotion-3.f, so it is easy to identify why the intrinsics functions fails to compile when using invalid types. So for example :
  - "LOGICAL a_i" with that notation would be "a_l" 
  - "CHARACTER a_r" would be "CHARACTER a_c".

Reply:
No problem.  I'll adjust the test.

Reply:
Thank you Jeff, looking forward for the updated patch.